### PR TITLE
Fix: Setting adapter does not take effect && Log processor.

### PR DESCRIPTION
**Pull request by Player626**
- Fix: Setting adapter does not take effect
- Fix: Log processor.

### CODE REVIEW

File: src/Config.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ class Config
     /**
      * @var string
      */
-    protected $logger_class = 'EasySwoole\Permission\Logger';
+    protected $logger_class = Logger::class;
```
Comment (Player626) [inline src/Config.php:43]: Sorry, I didn't see this `pr`.
It's not reasonable to write `classname` in this place, but now that the version has been released, let's do it.

File: src/Casbin.php [hunk 2]
```diff
@@ -116,15 +116,15 @@ public function __construct(?Config $config = null)
 
         $this->config = $config;
 
-        if (!$config->getAdapter() instanceof Adapter) {
-            $this->adapter = new DatabaseAdapter();
-        }
-
-        if ($logger = $this->config->getLoggerClass() ?? 'EasySwoole\Permission\Logger') {
-            if (class_exists($logger)) {
-                $logger = $logger::getInstance();
```
Comment (Player626) [inline src/Casbin.php:125]: It is wrong to call the `getInstance` method in this place. The log processor set by the user is not necessarily a singleton.

**Comment by hsluoyz** [comment]
@gaobinzhan PR title should be more than "Fix" only.

**Comment by hsluoyz** [comment]
@techoner plz review.

**Pull request merged**